codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttPublishVariableHeader.java

```diff
@@ -24,27 +24,35 @@
 public final class MqttPublishVariableHeader {
 
     private final String topicName;
-    private final int messageId;
+    private final int packetId;
 
-    public MqttPublishVariableHeader(String topicName, int messageId) {
+    public MqttPublishVariableHeader(String topicName, int packetId) {
         this.topicName = topicName;
-        this.messageId = messageId;
+        this.packetId = packetId;
     }
 
     public String topicName() {
         return topicName;
     }
 
+    /**
+     * @deprecated Use {@link #packetId()} instead.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)